### ISSUE TITLE
In practice, first recorded block might not be block 1

### ISSUE THREAD

**Issue by shippy** [YaleDecisionNeuro/PsychTaskFramework#96]
In many places, PTF expects that the index of the current block is `numFinishedBlocks + 1`. That's where it proceeds to save the recorded data. But for various reasons, that might not be the block that was run -- e.g. in `RA`, the practice runs blocks 6 and 7 to do one block from each domain.

In practical terms, the best practice would be to remove the blocks that don't get run; that makes things work, and doesn't require additional programmatic checks. (Sure, `runNthBlock` can check if `n > Data.numFinishedBlocks`, and we could pass current `blockIdx` to `runBlock` in `blockSettings` somewhere, but the only thing this messes up is the output of practice runs that doesn't get saved anywhere.)

Here's a proof of concept for the block removal:

```matlab
numBlocks = 8; % retrieved earlier
practiceBlockIdx = 6:7;
allBlocks = 1:numBlocks;
Data.blocks(setdiff(allBlocks, practiceBlockIdx)) = [];
for blockIdx = 1:numel(practiceBlockIdx)
  % run the two blocks, which will now be correctly recorded
end
```

**Comment by shippy** [comment]
Actually, easier:

```matlab
Data.blocks = Data.blocks(practiceBlocks);
```

**Comment by shippy** [comment]
I think this was fixed in bdd59603bd04d3c88d9b108590e64a5651e14999; closing.
